cime_config/tests.py

```diff
@@ -244,6 +245,8 @@
         "tests"   : (
             "SMS_Ld1.ne30pg2_r05_EC30to60E2r2.WCYCL1850.allactive-wcprod",
             "SMS_Ld1.ne30pg2_EC30to60E2r2.WCYCL1850.allactive-wcprod",
+            "SMS_D_Ld1.ne30pg2_EC30to60E2r2.WCYCLSSP370.allactive-wcprodssp",
```

Comment:
Why do we have the same test `SMS_D_Ld1.ne30pg2_EC30to60E2r2.WCYCLSSP370.allactive-wcprodssp` in both the production and the integration suites? Is it to ensure the machine coverage for production environment? May be we can change this test to make it more like "production style":
1. Remove `_D` to make it non-debug
2. Increase the number to days or remove `_Ld1` to run it for 5 days (default)

Reply:
The production suite is run on more platforms.  It should be a subset of integration.

Reply:
Actually no its not a subset.  But it is production cases which are expensive so its a short list.

Reply:
Yes, I think we can run this configuration without `_D` in the production suite, since we are running it with `_D` (debug) in the integration suite.

Reply:
You should also add this test to the production suite in maint-2.0

Reply:
Thanks for careful review, @singhbalwinder , @rljacob . I am going to remove _D in the production tests. The current PR is going to maint-2.0 (by default, as the branch was based off maint-2.0). I intend to use the same branch to create a new PR for master. Any concern on doing this? All updates in maint-2.0 since the branch was created are also in master.

Reply:
That should be fine.